.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
```

Comment:
I think instead of creating this file you should add it's config in the `pyproject.toml`

Reply:
done in 5eea4ad